spec/features/providers/ucas_alerts_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "rails_helper"
+
+feature "Edit UCAS email alerts", type: :feature do
+  let(:page) { PageObjects::Page::Organisations::UcasContactsAlerts.new }
+  let(:org_ucas_contacts_page) { PageObjects::Page::Organisations::UcasContacts.new }
+  let(:provider) { build(:provider, send_application_alerts: nil) }
+  let(:current_recruitment_cycle) { build :recruitment_cycle }
+
+  before do
+    stub_omniauth
+    stub_api_v2_resource(provider)
+    visit alerts_provider_ucas_contacts_path(provider.provider_code)
+  end
+
+  scenario "selecting and saving an option" do
+    expect(current_path).to eq alerts_provider_ucas_contacts_path(provider.provider_code)
+    expect(page.title).to have_content("Email alerts for new applications")
+    expect(page.main_heading).to have_content("Email alerts for new applications")
+    expect(page).to have_alerts_enabled_fields
+    expect(page.alerts_enabled_fields).to have_all
+    expect(page.alerts_enabled_fields).to have_none
+    expect(page.alerts_enabled_fields.all).not_to be_checked
+    expect(page.alerts_enabled_fields.none).not_to be_checked
+    set_alerts_request_stub_expectation("none")
+    page.alerts_enabled_fields.none.click
+    click_on "Save"
+    expect(org_ucas_contacts_page).to be_displayed
+    expect(org_ucas_contacts_page.flash).to have_content("Your changes have been saved")
```

Comment:
don't you need to check is the none aka `Don’t get an email` is showing?

Reply:
that's tested in the contacts page spec